CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 - Update `buf format -w` so that it does not touch files whose contents don't actually
   change. This eliminates noisy notifications to file-system-watcher tools that are
   watching the directory that contains proto sources.
+- Fixes issue where `buf lint` fails when the message is deeply nested.
```

Comment:
I'm not sure what this means.

Reply:
Updated in [2440513](https://github.com/bufbuild/buf/pull/3072/commits/24405130b09f52b599c6e03d9039e289104da4a7), I also noticed that it could happen in `buf breaking` as well.